### DISCUSSION TITLE
Edit rocks install path

### DISCUSSION THREAD

**Discussion by TheCogster409** [keplerproject/luarocks#1713] [Q&A]
I have my lua and luarocks install on a seprate drive from where the rocks are installed. How do I change this? i made a folder called rocks in my luarocks installation for everything to go.

I asked chatgpt and it told me to make a luarocks.cfg file in the same location as luarocks.exe with this stuff in it:
```cfg
rocks_trees = {
    { name = [[user]], root = [[C:\path\to\your\custom\user\rocks\directory]] },
    { name = [[system]], root = [[C:\path\to\your\custom\system\rocks\directory]] },
}
```

but it doesnt seem to work.

**Comment by Ra2-IFV** [comment]
You need this?
https://github.com/luarocks/luarocks/wiki/Config-file-format#locations
